packages/cbioportal-utils/src/hotspot/CancerHotspotsUtils.spec.ts

```diff
@@ -209,6 +376,77 @@ describe('CancerHotspotsUtils', () => {
         });
     });
 
+    describe('isRecurrentHotspot', () => {
+        it('checks if a mutation is a hotspot mutation by isRecurrentHotspot()', () => {
+            assert.isTrue(
+                isRecurrentHotspot(hotspotMutation1, hotspotIndex),
+                'TP53 R273C should be a recurrent hotspot mutation.'
+            );
+
+            assert.isTrue(
+                isRecurrentHotspot(hotspotMutation2, hotspotIndex),
+                'TP53 R273A should be a recurrent hotspot mutation.'
+            );
+
+            assert.isTrue(
+                isRecurrentHotspot(hotspotMutation3, hotspotIndex),
+                'PIK3CA R38H should be a recurrent hotspot mutation.'
+            );
+
+            assert.isFalse(
+                isRecurrentHotspot(notHotspotMutation, hotspotIndex),
+                'SMURF1 R101F should not be a recurrent hotspot mutation.'
+            );
+
+            assert.isFalse(
+                isRecurrentHotspot(hotspot3dMutation, hotspotIndex),
+                'SMURF1 R101N should not be a recurrent hotspot mutation.'
+            );
+        });
+    });
+
+    describe('is3dHotspot', () => {
+        it('checks if a mutation is a hotspot mutation by is3dHotspot()', () => {
+            assert.isFalse(
+                is3dHotspot(notHotspotMutation, hotspot3dIndex),
+                'SMURF1 R101F should not be a 3d hotspot mutation.'
+            );
+
+            assert.isTrue(
+                is3dHotspot(hotspot3dMutation, hotspot3dIndex),
+                'SMURF1 R101N should be a 3d hotspot mutation.'
+            );
+        });
+    });
+
+    describe('isSpliceHotspot', () => {
+        it('checks if a mutation is a hotspot mutation by isSpliceHotspot()', () => {
+            assert.isFalse(
+                isSpliceHotspot(notHotspotMutation, spliceHotspotIndex),
+                'MET X1028_splice should not be a splice hotspot mutation.'
+            );
```

Comment:
maybe good to add MET X1010 as well as an expected hotspot in the tests. Just b/c the originally issue specifically mentions that one as the one missing. Good to proof that it's working in a test. Maybe you already have this somewhere but I didn't see it (I did see it in the test data)

Reply:
@inodb yes I have a test for it but got messes up with messages... let me update it